webhook-integrations/Runscope/notes.md

```diff
@@ -0,0 +1,29 @@
+# Runscope Integration Investigation
+
+* [Runscope](https://runscope.com) is a company that provides solutions for API Performance Testing.
+
+* Runscope helps a company to conduct API Performance tests on an independent, hassle-free platform.
+
+* Runscope already has integrations with Slack & generates notifications in Slack for the following cases:
+
+  1. Test Passed
+
+  2. Test Failed
+
+* [The Notification Model Image](https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/notification_model.PNG)
```

Comment:
Hey. The links to images are still absolute. They could be easily replaced with just the name of the image file because they are in the same directory. :smile: